MAINTAINERS.md

```diff
@@ -14,16 +14,9 @@
 |------|--------|------|
 | Tech Lead | Jamie Monserrate | [jamiemonserrate](https://github.com/jamiemonserrate) |
 | Tech Lead  | Shailesh Pant | [pshail](https://github.com/pshail) |
-| Eng Management | Gavin Lu |[luguang](https://github.com/luguang) |
 | Eng Management | Sudheesh Sudhakaran |[sudheesh-csp](https://github.com/sudheesh-csp) |
 | Eng Management | Vijayakumar Arumuga Nadar |[Vijayakumar0328](https://github.com/Vijayakumar0328) |
-| Product Management | Disha Chopra |[dishac](https://github.com/dishac) |
-| Engineering | Aleksandar Georgiev |[georgievaVMW](https://github.com/georgievaVMW) |
 | Engineering | Anusha Hegde |[anusha94](https://github.com/anusha94) |
```

Comment:
I'd rather have my name removed from the engineering list as I won't be able to actively contribute anymore.